packages/vaadin-virtual-list/README.md

```diff
@@ -0,0 +1,40 @@
+# Vaadin virtualizer
```

Comment:
This now needs to be `vaadin-virtual-list` and the README has to be updated accordingly.

Reply:
Done. Will write virtual-list readme in the scope of https://github.com/vaadin/web-components/issues/283